ntfs_fare_extension.md

```diff
@@ -13,25 +13,34 @@ The data format of the files are the same as the core [NTFS ones](./ntfs_fr.md).
 
 File | Constraint | Comment
 --- | --- | ---
-tickets.txt | Required | Contains tickets declaration.
-ticket_prices.txt | Required | Contains prices of a ticket.
-ticket_uses.txt | Required | Contains where the ticket can be used
-ticket_use_perimeters.txt | Required | Defines the networks and lines of a ticket_use
+tickets.txt | Optional | Contains tickets declaration.
+ticket_prices.txt | Optional | Contains prices of a ticket.
+ticket_uses.txt | Optional | Contains where the ticket can be used
+ticket_use_perimeters.txt | Optional | Defines the networks and lines of a ticket_use
 ticket_use_restrictions.txt | Optional | Defines a use of a ticket between zones or stop_areas
 
+A ticket definition requires to be valid :
+* a ticket
+* a ticket use
+* a ticket use perimeter
+A ticket price is strongly recommanded. If not privided, Navitia will provide a ticket without price 'or with a 0€ price).
```

Comment:
```suggestion
A ticket price is strongly recommended. If not provided, Navitia will provide a ticket without price (or with a null price).
```

Reply:
Thanks for the typos, it's fixed. For the 0€, it's the actual response of navitia if there is no price and the NTFS is provided by transit_model (backward compatibility of fares). Thus I'm in favor of keeping it like this